db/migrate/20211028231536_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :postsCounter
+      t.integer :posts_counter
```

Comment:
- [ ] [optional] It is highly recommended to not edit migration files, you should rather create another migration file in order to make the changes you want, so that your db will stay in a consistent state, otherwise you might get difficult situation in production

Reply:
Thank you for the recommendation. :rocket: I will make sure to create new migrations whenever there is a need to update a DB.